packages/simon/src/frontright-classic-cuton.js

```diff
@@ -25,24 +25,24 @@ export default (part) => {
   if (complete) {
     // Placket help lines
     paths.frontCenter = new Path().move(points.cfNeck).line(points.cfHem).attr('class', 'help')
+    if (!options.seperateButtonholePlacket) {
+      // Match lines are only displayed on attached plackets
+      paths.frontCenter
+        .attr('data-text', 'matchHere')
+        .attr('data-text-class', 'text-xs center')
+    }
     paths.placketInnerFold = new Path()
       .move(points.placketBottomIn)
       .line(points.placketTopIn)
       .attr('class', 'dotted')
-      .attr('data-text', 'matchHere')
-      .attr('data-text-class', 'text-xs center')
     paths.placketOuterFold = new Path()
       .move(points.placketTopOut)
       .line(points.placketBottomOut)
       .attr('class', 'dotted')
     macro('sprinkle', {
       snippet: 'notch',
       on: [
-        'placketTopIn',
-        'placketTopOut',
         'cfNeck',
-        'placketBottomIn',
-        'placketBottomOut',
```

Comment:
Does it make sense to remove these? Maybe these notches were being used for something other than matching the sides? If it does make sense, should I also remove the notches on the buttonhole placket?

Reply:
These notches are here to help people fold the placket during construction.

Reply:
Ok, I will revert these changes. I had the impression that notches were only used to match two pieces of fabric. That said, I have literally never seen a notch before this pattern.

Reply:
Added a commit reverting the notch removal.